### PR TITLE
Implement new payment method for PH channels: GCash

### PR DESCRIPTION
**Pull request by andykim**
## Description

`Implement payment method for PH channels: GCash`

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Cosmetic change (text changing or color changing)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code (if necessary), particularly in hard-to-understand areas
- [ ] My changes generate no new vulnerabilities
- [ ] I have added tests that prove my fix is effective or that my feature works

## Screenshot

`put screenshot here if there are any ui changes`

Demo here:
https://drive.google.com/file/d/1X8vQnyucZgE7lAbk-Ppv_xBtsgJneH5O/view?usp=sharing
https://drive.google.com/file/d/15-8elam_MRQe1vhlbQ0AsYZswVxdIefG/view?usp=sharing

For multishipping: https://drive.google.com/file/d/1OovNZqyboNnKGpj6RiUELdA0c4CFRP4_/view?usp=sharing

### CODE REVIEW

**Comment by candrasaputra** [comment]
Hi @andykim are you already try on multishipping flow? because multishipping is our common use case.

**Comment by andykim** [comment]
> Hi @andykim are you already try on multishipping flow? because multishipping is our common use case.

Hi @candrasaputra . Yes, I do for multi-shipping flow too

**Comment by candrasaputra** [comment]
Okay cool. @andykim

**Review by candrasaputra — APPROVED** [review]

**Review by yanoandri — APPROVED** [review]

File: Xendit/M2Invoice/etc/config.xml [hunk 1]
```diff
@@ -245,6 +245,19 @@
...
+                <payment_action>initialize</payment_action>
+                <model>Xendit\M2Invoice\Model\Payment\Gcash</model>
+                <title>GCASH</title>
+                <description>Pay for orders using GCash via Xendit</description>
+                <min_order_total>1</min_order_total>
+                <max_order_total>10000000</max_order_total>
+                <is_gateway>1</is_gateway>
+                <can_initialize>1</can_initialize>
+                <can_use_checkout>1</can_use_checkout>
+                <sort_order>17</sort_order>
```
Comment (IreneGohtami) [inline Xendit/M2Invoice/etc/config.xml:259]: could you change the sort_order for gcash? basically I wanna sort it alphabetically but grouped per type, so switch with paymaya.

**Pull request merged**